plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlLegacyConnectorTest.java

```diff
@@ -135,6 +135,14 @@ public void testRenameColumn()
                 .hasStackTraceContaining("RENAME COLUMN x TO before_y");
     }
 
+    @Override
+    public void testNativeQuerySelectFromNation()
+    {
+        // override because MySql legacy returns varchar(255)
```

Comment:
You can avoid override in such case with `io.trino.sql.query.QueryAssertions.QueryAssert#skippingTypesCheck`
(or by using `assertQuery("trino query", "h2 query")` which also skips type checks and does even more)